app/src/main/java/protect/card_locker/LoyaltyCardEditActivity.java

```diff
@@ -1336,27 +1352,54 @@ public void onDialogDismissed(int dialogId) {
     public static class DatePickerFragment extends DialogFragment
             implements DatePickerDialog.OnDateSetListener {
 
-        final Context context;
-        final EditText textFieldEdit;
+        private static final String TEXT_FIELD_TO_EDIT_ARGUMENT_KEY = "text_field_to_edit";
+        private static final String CURRENT_DATE_ARGUMENT_KEY = "current_date";
+        private static final String MIN_DATE_ARGUMENT_KEY = "min_date";
+        private static final String MAX_DATE_ARGUMENT_KEY = "max_date";
+        private static final String PICK_DATE_REQUEST_KEY = "pick_date_request_key";
```

Comment:
Any reason the value ends with "_key" unlike the others?

Reply:
Nothing, really. I'll just remove it.